pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -468,7 +479,22 @@ func (ctx *context) handleInboundResponse(response *Response) error {
 	}
 	dest := prepareDestination(conn.DIDDoc)
 
-	//TODO:Send ver key issue-299
-	sendVerKey := ""
+	sendVerKey, err := ctx.createSendVerKey()
+	if err != nil {
+		return fmt.Errorf("handle inbound response: %s", err)
+	}
+
 	return ctx.outboundDispatcher.Send(ack, sendVerKey, dest)
 }
+
+func (ctx *context) createSendVerKey() (string, error) {
+	didDoc, err := ctx.didWallet.CreateDID(didMethod)
```

Comment:
you should use the same did for populating exchange request

Reply:
done